src/cmd/run.js

```diff
@@ -303,7 +339,7 @@ export type ExtensionRunnerParams = {|
   startUrl?: string | Array<string>,
 |};
 
-export class ExtensionRunner {
+export class DefaultExtensionRunner {
```

Comment:
can we leave this named as `ExtensionRunner` instead of renaming it to `DefaultExtensionRunner`?